Source/Core/ServiceTask.swift

```diff
@@ -107,6 +107,12 @@ import Foundation
 
 extension ServiceTask {
     /// TODO: Needs docs
```

Comment:
Can you please add documentation for this method?

Reply:
Suggested documentation:
```
/**
 Sets a boolean that indicates whether the receiver should use the default 
 cookie handling for the request.
 
 - parameter handle: true if the receiver should use the default cookie 
   handling for the request, false otherwise. The default is true.
 - returns: Self instance to support chaining.
*/
```